components/sensor/ld2410.rst

```diff
@@ -9,8 +9,12 @@ Component/Hub
 -------------
 .. _ld2410-component:
 
-The ``ld2410`` sensor platform allows you to use HI-LINK LD2410 motion and presence sensor
-(`datasheet and user manual <https://drive.google.com/drive/folders/1p4dhbEJA3YubyIjIIC7wwVsSo8x29Fq-?spm=a2g0o.detail.1000023.17.93465697yFwVxH>`__) with ESPHome.
+The ``ld2410`` sensor platform allows you to use HI-LINK LD2410 motion and presence sensors with ESPHome.
+There are three variants with similar communication protocols:
+
+- LD2410 (`datasheet and user manual <https://drive.google.com/drive/folders/1lCQv3mfHJ3XKXzweeHPFnJ_8_D_EWEKk>`__)
```

Comment:
It is odd to me these are Google Drive folders. Besides that, those folders are owned by a different owner compared what was in here before.

Can we make use of something else in this case? Are there other public resources we can link to?

Reply:
While I agree on your point, the official page from the manufacturer links to the google drive folder :facepalm: 

https://www.hlktech.net/index.php?id=988

![2024-11-11_14-25-25](https://github.com/user-attachments/assets/42f001b2-3b28-44a8-ae56-db9ecd17561f)

Reply:
lol OK... not way out for us. 🙈 

WHY!?! 😭

Reply:
Sorry I'm a bit late to the conversation :)
Just to add confirmation - indeed I got these GDrive links from the Hi-Link website.
This is from the product page:
![hlk](https://github.com/user-attachments/assets/7910ecaa-2c7c-4924-a277-af8f099865cf)